### DISCUSSION TITLE
Conscientious Contrarian Academic

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#262] [purpose heart prior; visioning 🔶]
Conscientious Contrarian is someone who finds value where others optimize for the wrong things. They map who needs X but gets Y because providers optimize for Z, then bridge this gap sustainably.

---

## On the Edge: The Art of Risking Everything Book by Nate Silver (chp.13)

Successful risk takers are conscientiously contrarian. They have theories about why and when the conventional wisdom is wrong. There is an often neglected distinction between independence and contrarianism. If I pick vanilla and you pick chocolate, but if you like chocolate better, you're being independent. If you pick chocolate because I pick vanilla, you're being contrarian.
Most people are pretty damned conformist. Humans are social animals, and rebellians are sometimes accused of being contrarian when they're just being independent. If I do the conventional thing 99% of the time, and you do it 85% of the time, you'll see rebellions by comparison, but you're still mostly going with the flow. But there are some parts of the river that have an explicit mandate to be contrarian. 
- Sports Betting is one example. If you just follow the public money, you'll lose the same 5% juice to the house everyone else loses. if not more than that, recreational sports bettors generally do slightly worse than if they just chose picks at random. So as a default, it's sometimes better to fade the public and make unpopular picks. 
- Hedge funds are another example. They charge investors exorbitant fees on the premise of achieving excess returns above and beyond what any old earning index fund could get in the S&P500. The thing about hedge funds, it's the only industry in the world where not only do you have to be right, but everybody else also has to be wrong. 

Galen Hall, who earned more than $4 million from poker tournaments between 2011 and 2015 but nevertheless left to work for bridgewater associates. He was a bunch of recruited his friend selves there. Whereas if you're a doctor, if you go by the book, you do the accepted thing, you do the commission every single time you're a fucking good doctor, right? Hall is one of those people who's just good at everything. He's good at poker. He's good at finance. In 2023 he reached the semifinals in the world backgammon championship despite just having picked up the game. When during the 2022 World Series of Poker, there was a false report of a mass shooter on the les vegas strip, he broke through a ceiling event to lead people to a safe hiding place. With dark features, he's also extremely good looking. It turns out Hall does have some flaws, though. Bridgewater is obsessed with quantifying its trader's personalities and performance, assigning them baseball cards as though they're Major League shortstops. My organized and reliable rating was three out of 100 my rules following was a two out of 100, Hall said. the only person lower than me was my then boss. 

But that rebellious streak serves Hall well and his new firm, DFT, dark forest technologies, which he co founded, along with Jacob Klein, his expert Twitter boss, Klein got a one out of 100 in rules following. DFT engages in the strategy that [🗣️I'm going to dub "conscientious contrarianism". They make contrarian bets, but they always have a thesis for doing so. One that has nothing to do with the intrinsic qualities of the asset, but rather with the misaligned incentives of other participants in the market ](https://otter.ai/s/RTstqiLzS0a_qLzTGpucuQ?snpt=true). Every single thing we do, I can point to like here is a person who is doing a thing wrong. We build a map of all the players in the world who are trading for reasons other than they want to generate some alpha. Alpha refers to achieving excess returns. The EV maximizing goal of most active investors. You know, somebody got added to the S&P500. So now all the S&P500 ETFs out there have to buy this company, or to CEO of a startup. He's now a billionaire. He has 99% of his net worth in the company. He's allowed to sell his shares in the on this day, and he's probably going to sell a bunch. 
I like this strategy because it doesn't insult the intelligence of other players in the market. Sure, Hall is smart, but so are lots of people on Wall Street. In a game theory equilibrium, it's hard for anybody to have alpha, but not everybody is playing the same game. People have different incentives, and they may be following their incentives rationally, but it nevertheless creates profitable opportunities for DFT, which just wants to make money.

Maybe conscientious contrarianism resonates with me because it reminds me of what it's like to argue with people about politics on the internet. It might not appear this way when you look at my Twitter replies, but my goal, when I do this, in line with my background in forecasting, is to seek accuracy and hopefully be proven right by subsequent events. 
That's not to suggest my motives are pure, dispassionate, truth seeking. It's also fun to be proven right. That's why I sometimes get frustrated and even challenge people to make bets on political outcomes, to put their money where their mouth is, but other people are trying to argue for a cause, rally the troops, boost their profile, or seek out approval on social media. All of that is fine, but they have different objectives. It's easier to achieve alpha at your objective when other people are playing the same game. This also helps to explain why so many of her academic colleagues dismissed Karikó's brilliant idea. Her mRNA vaccines were a high upside, but high risk bet that might take years to develop. Even Karikó thought they might only come to fruition after her death. 

Silicon Valley is specifically designed to make these sorts of bets, [🗣️but academia, isn't. "just the opposite". Karikó said it tends to optimize for conventional ideas that have a high likelihood of getting the grant proposal accepted but not all that much payoff.](https://otter.ai/s/5eHPNpz8Q3G8S6Yy399Ipw?snpt=true) 

[🗣️Look for flaws in people's incentives rather than their intelligence and then seek out a place where your own incentives are well aligned with your goals.](https://otter.ai/s/4a94dm2ZQg6exnhoXFl7Rw?snpt=true)

---
[Dark forest capital management](https://www.preqin.com/data/profile/fund-manager/dark-forest-capital-management/394012)
> The firm's objective is to generate risk-adjusted returns by employing a global, market-neutral, long-short equity strategy that seeks to apply a systematic investment process to combine long/short equity signals from a wide range of specialized structural flows and macro information in a self-consistent, non-linear manner.

**Comment by hyunjimoon** [comment]
angie's reflection:

## incentive aligning mechanisms for co-venturing
1. incumbent and entrant 
- giving incumbent (`sail`), entrant (`nail`)'s equity can incentivize both. `sail` can do nail in sail, `nail` can scale. this s-curve transition increases overall performance
- e.g. ibm and microsoft, nokia and apple, stan and gen (?)

2. investor and founder
  https://github.com/Data4DM/BayesSD/discussions/257#discussioncomment-10843622

## applying conscientious contrarianism to my situation
is academia an environment where my incentive align with my goal? 

which is more likely between academia change its incentive or i change my goal?

are academics playing the same game? can they? 

I can try analyzing academics with 

| Component | Description |
|-----------|-------------|
| 🤔 Contrarian Thinking | Theories challenging conventional wisdom |
| 🎯 Incentive Analysis | Identifying misaligned motivations of others |
| 🧠 Rational Thesis | Well-reasoned explanations for contrarian bets |
| 💹 Market Inefficiency Exploitation | Capitalizing on forced suboptimal actions |
| 🔄 Systematic Approach | Structured process for opportunity identification |
| 🎭 Goal Alignment | Seeking environments matching personal incentives |
| 🕰️ Long-Term Perspective | Pursuing ideas with delayed payoffs |


## language (conscientious contrarian) helped me understand my attraction to charlie, vikash, scott
using [becoming cca cld](https://claude.ai/chat/bcaf561b-afcf-4d70-a4a9-755db9759e3b)

| Component           | Charlie Fine                                                                                                                                                                      | Scott Stern                                                                                                                                                                                               | Vikash Mansinghka                                                                                                                                                                                                                   | Angie Moon                                                                                                                                                                                                                                                                               |
| ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Key Contrarian Idea | "Nail it, scale it, sail it" framework for dynamic capability based entrepreneurial operations                                                                                    | - Bayesian approach to entrepreneurship and innovation economics<br>- role of scientific openness in scientific research                                                                                  | Probabilistic programming as alternative to deep learning for AI that understands the world like humans do                                                                                                                          | (to be enhanced) Conversational inference of startup valuation and equity allocation using language model and probabilistic programming                                                                                                                                                  |
| Toolbox             | - Value chain model (supplier-oem-distributor)<br>- Double helix between modular and integral<br>- Clockspeed across value chain (+bull whip)<br>- Gear model<br>- Triple s-curve | - Entrepreneurial compass<br>- Map for Innovation ecosystem across U.S. with Quality Measure <br>- Idea Production Function $\dot{A} = f(L_A, K_A, A; Z_A, δ)$ where $A, \dot{A}$ : knowledge stock, flow | - Gen probabilistic programming platform<br>- Rational meaning construction<br>- Automated differentiation of expected values (ADEV) optimization algorithm<br>- SPPL: Probabilistic Programming with Fast Exact Symbolic Inference | - Probabilistic programs with  equity valuation game rule world model<br>- Language model probabilistic programs to answer user's natural language question <br>- ADEV-based queries for equity allocation optimization<br>- Rational meaning construction of  SAFE, term sheet document |
| Long-Term Vision    | Sustainable entrepreneurial success through adaptive operations and understanding of industry evolution                                                                           | Long-term economic growth through innovation; Alignment of economic policies with innovation goals                                                                                                        | Human-like reasoning in AI for complex, open-ended environments; unifying different approaches to meaning in language and thought                                                                                                   | AI-assisted, rational design of startup financial structures that amplify cooperation among stakeholders                                                                                                                                                                                 |

**Reply by hyunjimoon** [reply]
[nate silver's talk on the edge](https://www.youtube.com/watch?time_continue=1&v=3aTB5mW3_A4&embeds_referring_euri=https%3A%2F%2Fwww.perplexity.ai%2F&embeds_referring_origin=https%3A%2F%2Fwww.perplexity.ai&source_ve_path=Mjg2NjY)

**Reply by hyunjimoon** [reply]
meaning of 
> In a game theory equilibrium, it's hard for anybody to have alpha, but not everybody is playing the same game. People have different incentives, and they may be following their incentives rationally, but it nevertheless creates profitable opportunities for DFT, which just wants to make money.

EVERYONE IS PLAYING A DIFFERENT GAME. It's like a poker game where one player must show their cards (regulatory requirements), another must fold at certain times (risk management), while you're the only one actually trying to win money. Even if everyone plays "perfectly" given their constraints, you have an edge. Or halloween treat trade where A is allergic to chocolates, B has small candy bag, and I'm the only one who's optimizing for yummy candies. 
 
- Conscientious: DFT doesn't assume others are dumb—they understand everyone's constraints
- Contrarian: While others assume efficient markets mean no opportunities, DFT sees opportunities in the very structure of the market

**Reply by tomfid** [reply]
I think there's a missing piece here. People aren't "playing given their constraints" (or at least legal constraints). Many are simply cheating by trading on insider information.

For example, a bunch of people in congress are making impossible returns in the vicinity of 100%/year. https://unusualwhales.com/congress-trading-report-2024 . That's not some arbitrage on clever assessment of traders' constraints. It's just insider trading (or asymmetric information if you want to sugar coat it).

**Comment by hyunjimoon** [comment]
## ALGORITHM for 
implementing conscientious contrarian with Zizek's 👀looking awry and Bayes' 🪙exchangeability tool

#🥊A. Andrew Gelman’s paradox about treating some events as purely random (like coin flips) vs. completely ignorant events (like who wins a boxer vs. wrestler match).
#👓B. The Inspection Paradox, where “inspection from within” yields systematically biased or elongated intervals, misaligning inside vs. outside observer vantage points.
Quick Observations

Looking Awry in a Lacanian or Žižekian sense: things appear different once we realize how our vantage point is structured by desire or prior assumptions.
If you “look awry” at Gelman’s scenario: you might see that we treat the boxer/wrestler fight as if it had a precise probability (like 0.5) even though that’s an artifact of total ignorance. The paradox arises when we conflate “no knowledge” with “precise knowledge of randomness.”
For the Inspection Paradox, “looking awry” highlights that an inside observer’s viewpoint systematically distorts the distribution of intervals—just like how a subject’s desire or vantage can distort the “true” shape of reality.
Both paradoxes show how the viewpoint or assumptions create illusions or contradictions, reminiscent of how “looking awry” reveals aspects hidden in a “straight-on” view. In other words, the phenomenon that seems random or paradoxical might be a byproduct of how we define or measure the problem.

**Comment by hyunjimoon** [comment]
### phil anderson was post-rationalizing contrarian
<img width="300" alt="image" src="https://github.com/user-attachments/assets/53f571ef-39e0-47be-bcaa-8cc5377fa1de" />

phil avoided "popular mathematical" when choosing advisor and i think avoiding popular came first then "mathematical" is post-rationalized.

### potential connection with Pauli exclusion principle (of ensuring population diversity!)
> Just as the Pauli exclusion principle shows that electrons must occupy different quantum states, contrarian innovators like Anderson and the Wright Brothers advance by taking distinct paths - Anderson deliberately avoided the popular mathematical physics research under Schwinger that attracted his peers, while the Wright Brothers pursued pilot control over automatic stability. However, there's a key difference: Anderson's contrarianism was a conscious choice to swim "upstream" against the crowd, while the Wright Brothers' approach emerged from their technical judgment about what would work best.

<img width="300" alt="image" src="https://github.com/user-attachments/assets/55d30dc3-6e81-454e-a8ab-6d70fb4f3078" />

---

[angie's pick from mind over matter.pdf](https://github.com/user-attachments/files/18548454/angie.s.pick.from.mind.over.matter.pdf) shows other examples of phil being an rival with thomos kuhn, his contrarian nature manifested when choosing mentors, his advisor having reputation for being kind, him being a purposeful persuader, interaction with Pauli.

---
i learned from chatting with luca that `conditional distribution` is key in contrarian. e.g. disagreement in `how` given agreed `what`. 

i asked 
> if i truly believe being a contrarian is beneficial for society, does your theory endorse me to be contrarian for the sake of being contrarian? interesting talk about innovation and selective pressure: https://www.youtube.com/watch?v=xn29slfvHvk&ab_channel=VivaTechnology

luca answered
> Well it isn’t a given that being contrarian is beneficial for society so no

I asked 
> would proving “diversity of human species =beneficial to human species given uncertain environment” be enough to prove “having contrarians is beneficial for society”?

luca answered
> What I would say is that making welfare statements when beliefs are heterogeneous is very hard. Mainstream people who think contrarians are wrong will always want less diversity of thought (and contrarians as well, since they think mainstream are wrong)

[Brunnermeir14_welfare_dist_belief.pdf](https://github.com/user-attachments/files/18548474/Brunnermeir14_welfare_dist_belief.pdf)'s abstract

> This article proposes a welfare criterion for economies in which agents have heterogeneously distorted beliefs. Instead of taking a stand on whose belief is correct, our criterion asserts that an allocation is belief-neutral efficient (inefficient) if it is efficient (inefficient) under any convex combination of agents’ beliefs. Although this criterion gives an incomplete ranking of social allocations,
it can identify positive- and negative-sum speculation driven by conflicting beliefs in a broad range of economic environments.

**Comment by hyunjimoon** [comment]
using [entrepreneurial value creation framework cld](https://claude.ai/chat/1f5da312-34d0-4f5e-9508-03d2e5008fb6)

<html><head></head><body><h2>Conscientious Contrarian Examples Summary</h2>

Example | Actor N (Need X, Get Y) | Actor S (Serve N, Provide Z) | Actor C (Contrarian) | Misalignment | Contrarian Solution | Value Created
-- | -- | -- | -- | -- | -- | --
DFT Trading | Retail investors<br>Need: Alpha/returns | Institutional traders<br>Optimize for: Mandates, compliance | DFT (Hall & Klein)<br>Map forced trades | B trades for rules (ETF rebalancing, CEO diversification), not profit | Exploit predictable "dumb money" flows | Profit from structural inefficiencies
mRNA Vaccines | Humanity<br>Need: Innovative vaccines | Academia<br>Optimize for: Grants, safe publications | Karikó<br>Persist with risky research | B rewards incremental, fundable work over breakthroughs | Continue high-risk research outside traditional incentives | Revolutionary vaccine technology
NextOpt Analytics | Small businesses<br>Need: Affordable analytics | Big consulting firms<br>Optimize for: $1M+ contracts | You/NextOpt<br>Build for SMEs | B's cost structure requires large deals, ignores small clients | Create lightweight, scalable analytics platform | $85k grant, democratized analytics for Korean SMEs
Entrepreneurship Research | Entrepreneurs<br>Need: Practical decision tools | Academia<br>Optimize for: Data-rich VC studies | You (researcher)<br>Study actual founders | B studies where data exists (VCs), not where need is (entrepreneurs) | Develop STRAP framework from entrepreneur perspective | Bridge theory-practice gap with actionable tools

<h3>Key Pattern</h3>
<p>In each case, Actor B isn't wrong or incompetent—they're rationally following their incentives. The opportunity exists because those incentives don't align with Actor A's actual needs. The conscientious contrarian succeeds by understanding both perspectives and creating value in the gap.</p></body></html>

----

🟨 Introduction (10s)
Did you love playing “Hidden Picture” as a kid? It’s the game where whoever spots what nobody else sees first wins! I’m Angie—I study Bayesian entrepreneurship at MIT and I’m the founder of NextOpt. The secret to my research and my business? That very same game.

🟩 Success 1 — The Neighborhood Chicken Shop (10s)
The owner of a logistics company sighed, “Data consulting is too expensive!” Consulting firms only work with big corporations—like hiring a hotel chef to cook for a corner chicken shop. So I built an analytics service just for small and mid-sized businesses—and landed ₩100 million in government support.

🟩 Success 2 — Mandatory Buyers (10s)
Just as students are required to buy milk in the cafeteria, there are institutional investors who must buy stocks by regulation. By modeling this “forced demand” probabilistically, you can place bets before prices move.

🟩 Success 3 — Data Outside the Streetlight (10s)
You’ve seen someone who dropped their keys but only looks under the streetlight, right? Traditional entrepreneurship research focuses on the “well-lit” investor data. I dive into the darker corners of startup life to uncover the true distribution of founders’ decisions.

🟥 Conclusion — How to Play Hidden-Picture in Entrepreneurship (10s)
The recipe has three steps:
1️⃣ 🔍 **Observe** — Find who’s struggling
2️⃣ ❓ **Question** — Why has nobody helped?
3️⃣ 💰 **Execute** — Model that gap and quantify its value

There are hidden pictures all around you. Spot them first—think like an entrepreneur!
